dirs/dirs.go

```diff
@@ -330,7 +334,13 @@ func SetRootDir(rootdir string) {
 	SnapServicesDir = filepath.Join(rootdir, "/etc/systemd/system")
 	SnapUserServicesDir = filepath.Join(rootdir, "/etc/systemd/user")
 	SnapSystemdConfDir = SnapSystemdConfDirUnder(rootdir)
-	SnapBusPolicyDir = filepath.Join(rootdir, "/etc/dbus-1/system.d")
+
+	SnapDBusSystemPolicyDir = filepath.Join(rootdir, "/etc/dbus-1/system.d")
+	SnapDBusSessionPolicyDir = filepath.Join(rootdir, "/etc/dbus-1/session.d")
+	// Use 'dbus/services' and `dbus/system-services' to mirror
+	// '/usr/share/dbus-1' hierarchy.
+	SnapDBusSessionServicesDir = filepath.Join(rootdir, snappyDir, "dbus", "services")
```

Comment:
Hmm, could those be called `session-services`? I think it looks odd, especially right next to system services below.

Reply:
As the comment says, I decided to mirror the directory hierarchy found in `/usr/share/dbus-1`.  If someone is unfamiliar with D-Bus, `session-services` might be more descriptive.  For someone who is familiar though, it seems like a point of friction.

Reply:
I still think if mirroring is what we are going for then s/dbus/dbus-1/